types/osx-notifier/osx-notifier-tests.ts

```diff
@@ -0,0 +1,18 @@
+// Tests for osx-notifier.d.ts
+// Project: https://www.npmjs.com/package/osx-notifier
+// Definitions by: ZongJing Lu <https://github.com/SONIC3D>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Tests taken from documentation samples.
+
+///<reference path="../osx-notifier/osx-notifier.d.ts" />
```

Comment:
Reference should be unnecessary

Reply:
Thanks, I will modify it and resubmit later.